File: Answers/05_if/e06_resistors.py
# Získanie odporov od používateľa
R1 = int(input("Zadajte hodnotu odporu R1: "))
R2 = int(input("Zadajte hodnotu odporu R2: "))

# Výpis výsledkov
print(f"Výsledný odpor pri sériovom zapojení je: {R1 + R2}")
print(f"Výsledný odpor pri paralelnom zapojení je: {(R1 * R2) / (R1 + R2)}")

File: Answers/05_if/e06_resistors.cs
using System;

class Program
{
    static void Main()
    {
        int R1 = int.Parse(Console.ReadLine());
        int R2 = int.Parse(Console.ReadLine());

        Console.WriteLine($"Výsledný odpor pri sériovom zapojení je: {R1 + R2}");
        Console.WriteLine($"Výsledný odpor pri paralelnom zapojení je: {(R1 * R2) / (double)(R1 + R2)}");
    }
}
